src/index.html

```diff
@@ -1,17 +1,111 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="input1">
+          <label for="surname">Surname:</label>
+          <input type="text" autocomplete="off"
+          id="surname" required> <br>
+        </div>
+        <div class="input">
+          <label for="name">Name:</label>
+          <input type="text" autocomplete="off"
+          id="name"> <br>
+        </div>
+        <div class="input">
+          <label for="age">How old are You?</label>
+          <input type="number" id="age"
+          max="100" min="10"
+          value="12"> <br>
```

Comment:
```suggestion
          <input
            type="number"
            id="age"
            max="100"
            min="10"
            value="12"
          >
```
fix attribute indentations in all cases